mlflow/models/utils.py

```diff
@@ -104,3 +105,41 @@ def save(self, parent_dir_path: str):
         """Save the example as json at ``parent_dir_path``/`self.info['artifact_path']`.  """
         with open(os.path.join(parent_dir_path, self.info["artifact_path"]), "w") as f:
             json.dump(self.data, f, cls=NumpyEncoder)
+
+
+def save_example(mlflow_model: Model, input_example: ModelInputExample, path: str):
```

Comment:
Nit, should these be private methods (prefixed with `_`)? We've run into issues in the past where undocumented but public APIs (e.g. APIs that aren't prefixed with underscores) are assumed to be stable / meant for public use

Reply:
yes that makes sense.